tests/python/gaia-ui-tests/gaiatest/tests/unit/manifest.ini

```diff
@@ -21,6 +21,9 @@ lan = true
 [test_contacts.py]
 [test_file_manager.py]
 [test_kill.py]
+[test_kill_multiple.py]
+# Bug 1125759 - Intermitent unit test TestKill_test_kill_multiple
+expected = fail
```

Comment:
If this is intermittent then we shouldn't mark it as expected to fail, we should disable it.